packages/app-builder/src/components/Cases/Filters/CasesFiltersContext.tsx

```diff
@@ -13,6 +13,7 @@ export const casesFiltersSchema = z.object({
   statuses: z.array(z.enum(caseStatuses)).optional(),
   dateRange: dateRangeSchema.optional(),
   name: z.string().optional(),
+  snoozed: z.union([z.literal('true'), z.literal('false')]).optional(),
```

Comment:
suggestion: Maybe we can use an enum here and transform it to a boolean.
```suggestion
  snoozed: z
    .enum(['true', 'false'])
    .transform((val) => val === 'true')
    .optional(),
```

This way we can setup the CasesFiltersForm.snoozed as an optional boolean.
WDYT ?

Reply:
The problem here is that the `parseSafeQuery` fucntion decode Search params in query string, and the `qs` library set boolean as `snoozed=true` or `snoozed=false`.
That's why I had to do this gymnastic, else we should use different schema for the same data.
In fact that why I posted my message on Slack, here is a quick win, but we need to look this part of the codebase more thoroughly in order to fix those issues about encoding & decoding query string search params